apps/cli/src/__tests__/__examples__/bug-report-1.ts

```diff
@@ -1,6 +1,6 @@
 import { assign, createMachine } from 'xstate';
 
-type Data = {};
+type Data = unknown;
```

Comment:
With the configuration out of the way, the bulk of the work here was resolving the lint errors that followed.

Some errors I was able to resolve with small changes like this.